### PR TITLE
Merge of v2.0.0

### PR DESCRIPTION
**Pull request by kowalskiw**
Merge of changes leading to v2.0.0 release to master branch.

### CODE REVIEW

File: evac/__init.py__
Comment (kowalskiw) [inline evac/__init.py__:1]: required?

File: installer/config [hunk 1]
```diff
@@ -0,0 +1,25 @@
+AAMKS_PATH="/usr/local/aamks";
+AAMKS_SERVER="10.8.47.50";
```
Comment (kowalskiw) [inline installer/config:2]: IP to be removed

File: manager/results_collector.py
Comment (kowalskiw) [inline manager/results_collector.py:1]: @mmackiewicz97 - is this file required since `evac.worker.LocalResultsCollector` was introduced?

File: results/beck.py
Comment (kowalskiw) [inline results/beck.py:1]: `results.beck_new` replaced this code - removing it should be considered

File: results/comparative.py
Comment (kowalskiw) [inline results/comparative.py:1]: Replaced by `results.beck_new.Comparison` - to be removed?

File: montecarlo/cfast_mcarlo.py [hunk 2]
```diff
@@ -390,65 +191,72 @@ def _section_compa(self):# {{{
...
             collect.append("OFFSET = {}".format(round(v['face_offset']/100.0, 2)))           # COMPARTMENT1_OFFSET
             collect.append("FACE = '{}'".format(v['face']))                                  # FACE
-            collect.append("CRITERION = 'TIME' T = 0 F = {} /".format(how_much_open))         # OPEN CLOSE
-            txt.append(', '.join(str(i) for i in collect))
+            if v['type_tri'] == 'DOOR':
+                # open after 1st percentile of evacuees run
+                collect.append(f"CRITERION='TIME' T=0,{first_percentile},{first_percentile+1} F=0,0,{how_much_open} /")
+            else:
+                collect.append("CRITERION = 'TIME' T = 0 F = {} /".format(how_much_open))         # constatntly open
+            print(v['type_sec'], collect[:-1])
```
Comment (kowalskiw) [inline montecarlo/cfast_mcarlo.py:258]:
```suggestion
```

**Review by mmackiewicz97 — APPROVED** [review]

**Review by zarooba01 — APPROVED** [review]
Good work

**Comment by kowalskiw** [comment]
#242 still required to be resolved
